src/core/src/collection.rs

```diff
@@ -215,6 +216,12 @@ impl Collection {
         assert_eq!(sig.signatures.len(), 1);
         Ok(sig)
     }
+
+    pub fn select_picklist(&self, pick: &HashSet<(String, String)>) -> Self {
+        // @CTB: why do we need this clone here?
```

Comment:
because the signature for `select_picklist` below (in the manifest) is
```
    pub fn select_picklist(self, pick: &HashSet<(String, String)>) -> Self {
```
make it
```
    pub fn select_picklist(&self, pick: &HashSet<(String, String)>) -> Self {
```
instead, and you would not take ownership of the manifest, and so you don't need the clone